test/binary_to_text_test.cpp

```diff
@@ -189,6 +189,16 @@ INSTANTIATE_TEST_CASE_P(
          "%2 = OpTypeVector %1 4",
          spvtest::MakeInstruction(SpvOpConstant, {2, 3, 999}),
          "Type Id 2 is not a scalar numeric type"},
+        // Three tests to check the upper bits of literal numbers
+        {"%1 = OpTypeInt 8 1",
+        spvtest::MakeInstruction(SpvOpConstant, {1, 2, 0xABCDEF00}),
+        "Upper bits of literal number are invalid"},
+        {"%1 = OpTypeInt 16 1",
```

Comment:
You should also exercise cases between 33 and 63 bits.

Reply:
I wonder, is there is a better option than parameterized tests (TEST_P + INSTANTIATE) for cases like this, where we need the very same logic for all bit sizes?